mp4compose/src/main/java/com/daasuu/mp4compose/composer/Mp4ComposerEngine.java

```diff
@@ -281,8 +281,8 @@ private void runPipelines() {
                 if (progressCallback != null) {
                     progressCallback.onCurrentWrittenVideoTime(writtenPresentationVideoTimeUs);
```

Comment:
I think here we should catch original timestamp, not trimmed value, seems like that.

Reply:
Thanks !! please fix this...